dts/bindings/gpio/gpio-leds.yaml

```diff
@@ -6,6 +6,14 @@ title: GPIO LEDs
 description: GPIO LEDs parent node
 
 compatible: "gpio-leds"
+generate-child-initializer: true
```

Comment:
Since we now have a dedicated LED GPIO driver, we should think about moving this under `led` and also fixing the basic sample applications to use this new driver. This can be tackled later on also.

Reply:
Good point. I think this can be done after #21393 merge. Then I will know the final form of the DTS bindigs and geration. So I will do it while rebasing.